src/components/Leaderboard.js

```diff
@@ -0,0 +1,33 @@
+//@flow strict
+
+import * as React from 'react';
+
+// Leaderboard component. Displays top 10 players. The props takes a list of 10 players.
+// 
+// Prop Names:
+// leaderNames (Pass the list of 10 top players to display on the leaderboard)
+// 
+// EXAMPLE PROP USAGE = '<Leaderboard leaderNames={ ["John", "Bill", "Suck", "Big Suck", "HEhaw", "XXXXXfweckerXXXXX", "Yes", "2more", "9", "10haha"] } />'
+//
+// Needs to be updated to pass actual Users to it.
+class Leaderboard extends React.Component<Props> {

```

Comment:
Again, props need to be defined.
Here is a working example: https://github.com/RickLeinecker/Fortuna/blob/master/src/casus/blockBank/BlockBank.js

Reply:
Props have been created in both components.